include/seqan3/core/char_operations/predicate_detail.hpp

```diff
@@ -236,7 +236,7 @@ template <char_predicate... condition_ts>
 struct char_predicate_combiner : public char_predicate_base<char_predicate_combiner<condition_ts...>>
 {
     //!\brief The message representing the disjunction of the associated conditions.
-    static constexpr auto msg = detail::condition_message_v<'|', condition_ts...>;
+    inline static const std::string msg = detail::condition_message_v<'|', condition_ts...>;
```

Comment:
I wonder if using a `std::string_view` as the member is any better than `std::string`?

Reply:
That should not work, since we need the strings to use the append feature. They are not constant and thus the string view would have dangling pointers as being initialised with temporaries.